es6.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = {
+  extends: ['eslint-config-mm/es5'],
+  parserOptions: {
+    ecmaVersion: 6
+  // Can't use ES6 imports yet in node: https://github.com/nodejs/help/issues/53#issuecomment-165763446
+  // sourceType: 'module'
+  },
+  env: {
+    es6: true
+  },
+  rules: {
+    'arrow-parens': [2, 'always'],
+    'arrow-body-style': [2, 'always'],
+    'arrow-spacing': 2,
+    'no-const-assign': 2,
+    'no-confusing-arrow': 2,
+    'no-var': 1,
+    'object-shorthand': 1,
```

Comment:
Valid point. To clarify: do you suggest not enforcing anything here or to go further and actively discourage the use of object shorthands?

See http://eslint.org/docs/rules/object-shorthand#options